src/annotator/frame-observer.js

```diff
@@ -10,23 +12,25 @@ let difference = (arrayA, arrayB) => {
 export const DEBOUNCE_WAIT = 40;
 
 export default class FrameObserver {
-  constructor(target) {
-    this._target = target;
+  /**
+   * @param {Element} element - element to observe
```

Comment:
This comment could be clearer about why this `element` is needed, ie. that it is the root of the DOM subtree to watch for the addition and removal of `<iframe>` elements.